prow/config/jobs/tools-1.8.yaml

```diff
@@ -0,0 +1,71 @@
+branches:
+- release-1.8
+image: gcr.io/istio-testing/build-tools:release-1.8-2020-10-15T20-33-46
+jobs:
+- command:
+  - make
+  - build
+  name: build
+- command:
+  - make
+  - lint
+  name: lint
+- command:
+  - make
+  - test
+  name: test
+- command:
+  - make
+  - gen-check
+  name: gencheck
+- command:
+  - entrypoint
+  - make
+  - containers
+  name: containers
+  regex: docker/.+|cmd/.+
+  requirements:
+  - gcp
+  - docker
+  resources: build
+  type: postsubmit
+- command:
+  - entrypoint
+  - make
+  - containers-test
+  name: containers-test
+  regex: docker/.+|cmd/.+
+  requirements:
+  - gcp
+  - docker
+  resources: build
+  type: presubmit
+- command:
+  - entrypoint
+  - make
+  - -C
+  - perf_dashboard
+  - deploy
+  name: deploy-perf-dashboard
```

Comment:
this seems suspicious @carolynhu do we only want this job on master to avoid reverting changes to master?

Reply:
Would just need to add  disable_release_branching: true on the job in master so this doesn't happen in the future and remove here.